tracer.go

```diff
@@ -42,6 +42,7 @@ func NewTracer(ctx context.Context, cfg Config, payload json.RawMessage) (retTra
 	}
 	retTracer.eventData = data
 
+	// TODO: maybe switch WithSyncer to WithBatcher
```

Comment:
I think it's better to avoid "TODO" comments in the code.
If you think we should do it then we can talk about it and open a ticket for it.